lab_3_keywords_textrank/main.py

```diff
@@ -51,7 +53,12 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        if self._punctuation:
+            for char in text:
+                if char in self._punctuation:
+                    text = text.replace(char, ' ')
```

Comment:
can you explain why it is better to replace punctuation with a space instead of, for example, empty string?

Reply:
in fact, there could be found a couple of reasons why it is beneficial to replace punctuation marks with `' '` instead of `''`, as well as there are arguments for `''` instead of `' '`
So it is up to you, yet I expect that you are able to defend your choices